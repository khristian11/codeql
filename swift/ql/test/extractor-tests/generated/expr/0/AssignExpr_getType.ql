// generated by codegen/codegen.py
import codeql.swift.elements
import TestUtils

from AssignExpr x
where toBeTested(x)
select x, x.getType()
