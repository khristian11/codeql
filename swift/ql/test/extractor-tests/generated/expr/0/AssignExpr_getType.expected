| expressions.swift:32:1:32:5 |  ... = ... | () |
| expressions.swift:33:1:33:11 |  ... = ... | () |
| expressions.swift:34:1:34:11 |  ... = ... | () |
| expressions.swift:35:1:35:12 |  ... = ... | () |
| expressions.swift:36:1:36:12 |  ... = ... | () |
| expressions.swift:56:1:56:8 |  ... = ... | () |
| expressions.swift:75:5:75:10 |  ... = ... | () |
| expressions.swift:86:1:86:13 |  ... = ... | () |
| expressions.swift:133:3:133:22 |  ... = ... | () |
| expressions.swift:137:3:137:20 |  ... = ... | () |
| expressions.swift:139:3:139:11 |  ... = ... | () |
